composer.json

```diff
@@ -13,11 +13,10 @@
         }
     ],
     "require": {
-        "php": "^5.5 || ^7.0",
+        "php": "^5.5 || ^7.0 || ^8.0",
         "ext-dom": "*",
         "ext-libxml": "*",
-        "phpunit/phpunit": "^5.7.23 || ^6.4.3 || ^7.0 || ^8.0",
-        "phpunitgoodpractices/polyfill": "^1.1"
```

Comment:
no need for polyfill for sure ?

Reply:
Pretty much, all test are green https://travis-ci.org/github/PHP-CS-Fixer/phpunit-constraint-xmlmatchesxsd/builds/735014400 
, do you have any concerns about this change?

Reply:
it was added for some reason, i'm afraid that maybe tests in this repo are not enough and that it is unpredictable whether drop of it will break sth

Reply:
here, we are using the "Version" class - https://github.com/PHP-CS-Fixer/phpunit-constraint-xmlmatchesxsd/blob/master/src/Constraint/XmlMatchesXsd.php#L15

the polyfill was bringing it live, if it was missing: https://github.com/PHPUnitGoodPractices/polyfill/blob/master/src/aliases.php#L25

Reply:
let's see https://github.com/PHP-CS-Fixer/phpunit-constraint-xmlmatchesxsd/pull/13